app/actions/index.js

```diff
@@ -32,25 +30,25 @@ export function bootstrapAppData() {
     /**
      * Confirm data formatting then bootstrap the store
      */
-    function _validateEvt(evt) {
+    function validateEvt(evt) {
       return evt.data &&
-        evt.data.hasOwnProperty('data') &&
-        evt.data.hasOwnProperty('messageType');
+        evt.data.data &&
+        evt.data.messageType;
```

Comment:
The reason is because of the rule http://eslint.org/docs/rules/no-prototype-builtins

But I fixed it lazily, and forgot to go back and change it to use Object.